src/components/Tooltip.tsx

```diff
@@ -188,6 +188,7 @@ interface HelpTooltipProps {
   tooltip: ReactNode
   placement?: Placement
   offset?: number
+  iconSize?: FontAwesomeIconProps['size']
```

Comment:
I had to do something similar. Probably better to use the type:
here https://github.com/gnosis/dex-react/pull/1025/files#diff-493194965e4afe11bf4448365069f0aeR9